rbac/server/api/roles.py

```diff
@@ -171,6 +180,86 @@
     return json({"id": role_id, "description": role_description})
 
 
+@ROLES_BP.delete("api/roles/<role_id>")
+@authorized()
+async def delete_role(request, role_id):
+    """Delete a role by it's next_id.
```

Comment:
no worries, I'll change it. We should probably change both to be next_id's since they're internal next UUIDs or change the user next_id to user_id for consistency